packages/apollo-codegen-swift/src/helpers.ts

```diff
@@ -255,7 +255,9 @@ export class Helpers {
 
   dictionaryLiteralForFieldArguments(args: Argument[]): SwiftSource {
     function expressionFromValue(value: any): SwiftSource {
-      if (value.kind === "Variable") {
+      if (value === null) {
+        return swift`nil`;
```

Comment:
I'm assuming here that we want a literal `nil` in the result map instead of e.g. `NSNull()`, since the `ResultMap`'s type is `[String: Any?]`. That said, I haven't tested this change in the context of a full project, just the unit test as given.

@cltnschlosser I would appreciate it if you could test this with your full project.

Reply:
Yeah, I think `nil` makes sense. I didn’t actually use this in a project. I had discovered it when trying to workaround the other issue. I think we just need to verify that the resulting JSON has the null value. I can do that tomorrow.

Reply:
```
{
	"operationName": "Test",
	"query": "query Test {\n  employee(request: {id: null}) {\n    __typename\n    fullName\n  }\n}",
	"variables": null
}
```

Seems to work, and graphql-faker was fine with the request.